modules/jaggery-apps/user-dashboard/dashboard/samlsso.jag

```diff
@@ -18,10 +19,13 @@ else{
 
 
 if(request.getParameter("login") === 'true'){
+    authzFailure =request.getParameter("authz_failure");
 	encodedSAMLRequest = sso.client.getEncodedSAMLAuthRequest(issuer);
-	
+
 }else if(request.getParameter("logout") === 'true'){
 	var user = session.get(LOGGED_IN_USER);
+    authzFailure=request.getParameter("authz_failure");
+    application.put(AUTHZ_FAILURE,authzFailure);
```

Comment:
Looks like we are setting this as a application scope variable. Is this correct ? When do we clean this ? And where is it used ?